tests/test_downloader_handlers.py

```diff
@@ -442,6 +443,12 @@ class TestHttps10(TestHttp10):
     scheme = "https"
 
 
+class TestAiohttp(TestHttp):
```

Comment:
As the tested code requires the asyncio reactor its tests should be decorated with `@pytest.mark.only_asyncio`